packages/openapi-v3/src/json-to-schema.ts

```diff
@@ -0,0 +1,93 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {JsonDefinition} from '@loopback/repository-json-schema';
+import {SchemaObject, ExtensionValue} from '@loopback/openapi-v3-types';
+import * as _ from 'lodash';
+
+export function jsonToSchemaObject(jsonDef: JsonDefinition): SchemaObject {
```

Comment:
It would be great to have TSDocs for this function :).

Reply:
@b-admike could you give me a link of the commit you are reviewing lol, I am afraid you review the wrong commit, hope it's not the first commit! I didn't touch this line, it's just copied from `openapi-v2`

Reply:
oh okay! I just checked out this branch and was reviewing everything in the new package. What should I focus on?